documentation/src/main/docs/plantuml/disjoint.puml

```diff
@@ -0,0 +1,19 @@
+@startuml
```

Comment:
What's this file for?

Reply:
it's included in the doc to display a sequence diagram like this one: https://smallrye.io/smallrye-mutiny/assets/images/diag-36a779d2e869a0312e41e21ab27aab6a.png

Reply:
cheers